packages/kauri-wallet/src/app/api.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from "@angular/core";
+import { HttpClient, HttpHeaders } from "@angular/common/http";
+
+var headers_object = new HttpHeaders();
+    headers_object.append('Content-Type', 'application/json');
+    headers_object.append("Authorization", "Basic " + btoa("user:hi"));
+
+const httpOptions = {
+  headers: headers_object
+};
+
+@Injectable()
+export class ApiService {
+    constructor(private http: HttpClient) {}
+
+    getRawTransactions(addresses: any[]) {
+
```

Comment:
we should always type the arrays - so `string[]`

Reply:
@red010b37 updated 🌮